haskell/private/expansions.bzl

```diff
@@ -19,11 +19,12 @@ def expand_make_variables(name, ctx, strings, extra_label_attrs):
     strings = [ctx.expand_make_variables(name, str, {}) for str in strings]
     return strings
 
-def haskell_library_extra_label_attrs(attr):
-    return [
-        attr.srcs,
-        attr.extra_srcs,
-        attr.data,
-        attr.plugins,
-        attr.tools,
+def haskell_library_expand_make_variables(name, ctx, strings):
```

Comment:
Unrelated change but I did this to help me understand what exactly the difference was between the two invocations of `expand_make_variables` in `private/module.bzl`. I think it's an improvement overall so I think we should keep it.